core/domain/services/graphql/connections/RootQueryDatetimesConnection.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace EventEspresso\core\domain\services\graphql\connections;
+
+use EE_Base_Class;
+use EEM_Datetime;
+use EventEspresso\core\domain\services\graphql\connection_resolvers\DatetimeConnectionResolver;
+use EventEspresso\core\domain\services\graphql\connections\EventDatetimesConnection;
+use EventEspresso\core\services\graphql\connections\ConnectionInterface;
+use Exception;
+use WPGraphQL\Type\WPObjectType;
+
+/**
+ * Class RootQueryDatetimesConnection
+ * Description
+ *
+ * @package EventEspresso\core\domain\services\graphql\connections
+ * @author  Brent Christensen
+ * @since   $VID:$
+ */
+class RootQueryDatetimesConnection implements ConnectionInterface
+{
+
+    /**
+     * @var EEM_Datetime $model
+     */
+    protected $model;
+
+
+    /**
+     * DatetimeConnection constructor.
+     *
+     * @param EEM_Datetime               $model
+     */
+    public function __construct(EEM_Datetime $model)
+    {
+        $this->model = $model;
+    }
+
+
+    /**
+     * @return array
+     * @since $VID:$
+     */
+    public function config()
+    {
+        return [
+            'fromType'           => 'RootQuery',
+            'toType'             => 'Datetime',
+            'fromFieldName'      => 'datetimes',
+            'connectionTypeName' => 'RootQueryDatetimesConnection',
+            'connectionArgs'     => EventDatetimesConnection::get_connection_args(),
+            'resolve'            => [$this, 'resolveConnection'],
+            'resolveNode'        => [$this, 'resolveNode']
+        ];
+    }
+
+
+    /**
+     * @param $entity
+     * @param $args
+     * @param $context
+     * @param $info
+     * @return array
+     * @throws Exception
+     * @since $VID:$
+     */
+    public function resolveConnection($entity, $args, $context, $info)
+    {
+        $resolver = new DatetimeConnectionResolver($entity, $args, $context, $info);
+        return $resolver->get_connection();
+    }
+
+
+    /**
+     * @param $id
+     * @param $args
+     * @param $context
+     * @param $info
+     * @return EE_Base_Class
+     * @since $VID:$
+     */
+    public function resolveNode($id, $args, $context, $info)
+    {
+        return $this->model->get_one_by_ID($id);
+    }
+}
```

Comment:
also identical to logic in `core/domain/services/graphql/connections/RootQueryTicketsConnection.php` and therefore can be extracted